harmoni_actuators/harmoni_tts/config/tts_local_configuration.yaml

```diff
@@ -0,0 +1,12 @@
+tts:
+  default_param:
+    tts_config: "/root/harmoni_catkin_ws/model/tts/config.json"
+    tts_model: "/root/harmoni_catkin_ws/model/tts/tts_model.pth.tar"
+    vocoder_config: "/root/harmoni_catkin_ws/model/tts/config_vocoder.json"
+    vocoder_model: "/root/harmoni_catkin_ws/model/tts/vocoder_model.pth.tar"
+    scale_stats_path: "/root/harmoni_catkin_ws/src/HARMONI/harmoni_actuators/harmoni_tts/scale_stats.npy"
```

Comment:
To make things cleaner for the local install/non-docker folks and avoid absolute paths, I propose we add a model meta-package which would just serve as an easily findable folder for all AI models, e.g. `$(find harmoni_model)/.../your_model`

Reply:
That sounds like a good solution to the path issue. Would it be better to add that in this PR and update the other 2 services afterwards, or start a new PR later?

Reply:
I would do it the best way in any unmerged code (this PR) and then do another issue/PR for anything that is doing it the "old" way.